crates/core_arch/src/aarch64/neon/mod.rs

```diff
@@ -232,6 +246,23 @@ extern "C" {
     ) -> int8x16_t;
 }
 
+/// Add pairwise
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(addp))]
+pub unsafe fn vpaddq_u8(a: uint8x16_t, b: uint8x16_t) -> uint8x16_t {
+    vpaddq_u8_(a, b)
+}
+
+/// Polynomial multiply long
+//poly128_t vmull_p64 (poly64_t a, poly64_t b)
```

Comment:
Please remove this comment

Reply:
There are many unnecessary comments like this - please remove the others as well.

Reply:
removed